src/Tools/dotnet-openapi/src/DownloadFile.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.OpenApi.Tasks
+{
+    /// <summary>
+    /// Downloads a file.
+    /// </summary>
+    public static class DownloadFileExtensions
+    {
+        public static async Task<string> DownloadAsync(this HttpClient httpClient,
+            string uri)
+        {
+            if (string.IsNullOrEmpty(uri))
+            {
+                throw new ArgumentException("Uri parameter must not be null or empty.");
+            }
+
+            if (string.IsNullOrEmpty(uri))
```

Comment:
Huh? There is no DestinationPath on this method...

Reply:
In fact is this being used anywhere at all?

Reply:
It is not! good catch.